srcpkgs/geary/template

```diff
@@ -1,21 +1,19 @@
 # Template file for 'geary'
 pkgname=geary
-version=0.12.4
-revision=2
-build_style=cmake
-patch_args="-Np1"
-configure_args="-DNO_FATAL_WARNINGS=ON -DDESKTOP_UPDATE=OFF
- -DICON_UPDATE=OFF -DGSETTINGS_COMPILE=OFF
- -DGSETTINGS_COMPILE_IN_PLACE=OFF"
-hostmakedepends="desktop-file-utils gnome-doc-utils intltool iso-codes
- pkg-config"
-makedepends="enchant2-devel gcr-devel gmime-devel libcanberra-devel
- libgee08-devel libgirepository-devel libnotify-devel libsecret-devel
- libunique-devel vala-devel webkit2gtk-devel"
+version=0.13.0
+revision=1
+build_style=meson
+configure_args="-Dcontractor=true"
+hostmakedepends="desktop-file-utils intltool iso-codes pkg-config itstool
+ gobject-introspection"
+makedepends="enchant2-devel gcr-devel gmime-devel libcanberra-devel glib-devel
+ libgee08-devel libnotify-devel libsecret-devel vala-devel webkit2gtk-devel
```

Comment:
no need for vala-devel, put vala in hostmakedepends

Reply:
thanks, it was causing the cross issue